aligner/src/main/java/org/omegat/gui/align/AlignFilePickerController.java

```diff
@@ -499,16 +494,13 @@ public boolean accept(File f) {
      * <li>Target file path
      * </ol>
      *
-     * @param args
-     * @throws Exception
+     * @param args command arguments.
      */
-    public static void main(String[] args) throws Exception {
+    public static void main(String[] args) {
         System.setProperty("apple.laf.useScreenMenuBar", "true");
 
         Preferences.init();
         PluginUtils.loadPlugins(Collections.emptyMap());
-        Core.setFilterMaster(new FilterMaster(FilterMaster.createDefaultFiltersConfig()));
-        Core.setSegmenter(new Segmenter(SRX.getDefault()));
 
```

Comment:
This part should be reverted.

```suggestion
        Core.setFilterMaster(new FilterMaster(FilterMaster.createDefaultFiltersConfig()));
        Core.setSegmenter(new Segmenter(SRX.getDefault()));
```